scripts/README.md

```diff
@@ -11,3 +11,7 @@ Prints the summary of the migration to TypeScript.
 ### upload-package.sh
 
 Provides helper function for uploading package file to the remote machine using SSH and running specified command remotely.
+
+### load-latest-premium-docker-image.sh
```

Comment:
Maybe let's have it in alphabetical order - after `get-ts-migration-progress.sh` and before `upload-package.sh`.

Reply:
Another step towards perfection done!